sdk/tests/conformance/extensions/ext-blend-func-extended.html

```diff
@@ -0,0 +1,573 @@
+        shouldBe("gl.getParameter(gl.BLEND_DST_ALPHA)", `ext.${func}`);
+        wtu.glErrorShouldBe(gl, gl.NO_ERROR, "should be no errors");
+    }
+
+    const dbi = gl.getExtension("OES_draw_buffers_indexed");
+    if (!dbi) return;
+
+    debug("");
+    debug("Testing indexed blend state updates with SRC1 blend funcs");
+    for (const func of extFuncs) {
+        dbi.blendFunciOES(0, ext[func], gl.ONE);
+        wtu.glErrorShouldBe(gl, gl.NO_ERROR, `${func} accepted as blendFunciOES src`);
+        shouldBe("gl.getIndexedParameter(gl.BLEND_SRC_RGB, 0)", `ext.${func}`);
+        shouldBe("gl.getIndexedParameter(gl.BLEND_SRC_ALPHA, 0)", `ext.${func}`);
+        dbi.blendFunciOES(0, gl.ONE, ext[func]);
+        wtu.glErrorShouldBe(gl, gl.NO_ERROR, `${func} accepted as blendFunciOES dst`);
+        shouldBe("gl.getIndexedParameter(gl.BLEND_DST_RGB, 0)", `ext.${func}`);
+        shouldBe("gl.getIndexedParameter(gl.BLEND_DST_ALPHA, 0)", `ext.${func}`);
+        dbi.blendFuncSeparateiOES(0, ext[func], gl.ONE, gl.ONE, gl.ONE);
+        wtu.glErrorShouldBe(gl, gl.NO_ERROR, `${func} not accepted as blendFuncSeparateiOES srcRGB`);
+        shouldBe("gl.getIndexedParameter(gl.BLEND_SRC_RGB, 0)", `ext.${func}`);
+        dbi.blendFuncSeparateiOES(0, gl.ONE, ext[func], gl.ONE, gl.ONE);
+        wtu.glErrorShouldBe(gl, gl.NO_ERROR, `${func} not accepted as blendFuncSeparateiOES dstRGB`);
+        shouldBe("gl.getIndexedParameter(gl.BLEND_DST_RGB, 0)", `ext.${func}`);
+        dbi.blendFuncSeparateiOES(0, gl.ONE, gl.ONE, ext[func], gl.ONE);
+        wtu.glErrorShouldBe(gl, gl.NO_ERROR, `${func} not accepted as blendFuncSeparateiOES srcAlpha`);
+        shouldBe("gl.getIndexedParameter(gl.BLEND_SRC_ALPHA, 0)", `ext.${func}`);
+        dbi.blendFuncSeparateiOES(0, gl.ONE, gl.ONE, gl.ONE, ext[func]);
+        wtu.glErrorShouldBe(gl, gl.NO_ERROR, `${func} not accepted as blendFuncSeparateiOES dstAlpha`);
+        shouldBe("gl.getIndexedParameter(gl.BLEND_DST_ALPHA, 0)", `ext.${func}`);
+        wtu.glErrorShouldBe(gl, gl.NO_ERROR, "should be no errors");
+    }
+}
+
+function runShaderTests(extensionEnabled) {
+    debug("");
+    debug("Testing various shader compiles with extension " + (extensionEnabled ? "enabled" : "disabled"));
+
+    const shaderSets = [];
+
+    const macro100 = `precision mediump float;
+        void main() {
+        #ifdef GL_EXT_blend_func_extended
+            gl_FragColor = vec4(1.0, 0.0, 0.0, 1.0);
+        #else
+            #error no GL_EXT_blend_func_extended;
+        #endif
+        }`;
+    const macro300 = `#version 300 es
+        out mediump vec4 my_FragColor;
+        void main() {
+        #ifdef GL_EXT_blend_func_extended
+            my_FragColor = vec4(1.0, 0.0, 0.0, 1.0);
+        #else
+            #error no GL_EXT_blend_func_extended;
+        #endif
+        }`;
+    shaderSets.push([wtu.simpleVertexShader, macro100]);
+    if (contextVersion == 2) {
```

Comment:
It'd be fine to run the version 100 shader in the WebGL 2.0 test to make sure there are no regressions parsing the ESSL 100 form of the shaders.